scripts/new-change

```diff
@@ -61,6 +61,12 @@ type: {change_type}
 # or "AWS SDK for Java v2" if it's an SDK change to the core, runtime etc
 category: {category}
 
+# Your GitHub user name to be included in the CHANGELOG.
+# Every contribution counts and we would like to recognize
+# your contribution!
+# Leave it empty if you would prefer not to be mentioned.
+github_username: {github_username}
```

Comment:
Does {github_username} count as empty?

Reply:
Not sure what you mean, but it's empty by default.